### PR TITLE
Revert rutile yield from garnet processing to it's pre-nerf values

### PR DESCRIPTION
**Pull request by Eysla**
in #1927 titaniumtetrachloride was changed to need 3 rutile instead of 1. This was the result of new processing lines for bauxite and ilmenite that yielded much higher amounts. As a result of the changes to titaniumtetrachloride all other sources of rutile were nerfed by a factor of three. This change would return rutile to the same yield of titanium as before the titaniumtetrachloride rebalance.

### CODE REVIEW

**Comment by chochem** [comment]
I havent seen any ticket discussing this. the andradite recipe was added with no discussion either. Most of those got nerfed a bit so just adjusting for the chem balance change with 3x is not a great argument.

I think the bigger question is: do we even want titanium from just yellow garnet? or to put it differently, how terrible does the ratio have to be that we are ok with it.

**Comment by Eysla** [comment]
This recipe is extremely niche, in fact, I'm not sure that anyone even uses it as primary source of rutile other than those doing challange runs.

The issue is that that the changes made here #1970 made it so that the yield of titanium would be unaffected for those using bauxite/ilmenite with the new chem balancing. But what it fails to take into account is that rutile from all other sources would be greatly nerfed. It is a heavy investment that needs to be made to get sluice juice and all of its associated by-products that I believe self balances the already very low yield and doesn't need to be further decreased for the few that plan on utilizing it.

**Comment by chochem** [comment]
1970 was more specific. was not for every route.

Anyway, guess we should look at the actual numbers. Right now from either calcite or yellow garnet its 241 ores : 1 titanium. With this buff it would be 80 ores (calcite+yellow garnet) : 1 titanium. Of course one could also try to find andradite. the goal would probably be to get 9 titanium to then use crops/bees without any rocket?

Those numbers should be the starting point.

**Comment by Eysla** [comment]
> 1970 was more specific. was not for every route.

#1970 brought bauxite and ilmenite to their previous values after being effected by chem changes, which is specifically what this PR does AND has no adverse effects on other sources as #1927 does. 

> Anyway, guess we should look at the actual numbers. Right now from either calcite or yellow garnet its 241 ores : 1 titanium. With this buff it would be 80 ores (calcite+yellow garnet) : 1 titanium. Of course one could also try to find andradite. the goal would probably be to get 9 titanium to then use crops/bees without any rocket?

The title is misleading, this is not a buff - its just returning rutile via garnet back to its previous state. You're responsible for nerfing other sources of rutile, failed to justify the nerf to garnydinia in your original issue (#1927), and in fact stated intent opposing the change that was made (“player isn’t worse off”) (#1970).

> Those numbers should be the starting point.

Why should I have to justify the numbers that you changed? Why should we be even talking about a 300% nerf to something when you didn't even know that you nerfed.

**Comment by chochem** [comment]
my PRs you linked were very clear in what they did. On top of that there were tickets and multiple discussion in discord including.

why so defensive anyway, why are numbers a bad thing? Basically GlodBlock added some random lines with powerful outputs and its up to us to pick up the pieces, fix chem balance, and do game balance.

**Comment by Eysla** [comment]
> why so defensive anyway, why are numbers a bad thing? Basically GlodBlock added some random lines with powerful outputs and its up to us to pick up the pieces, fix chem balance, and do game balance.
 
 because you asked me to defend my PR?
 
 You have admitted that you didn't know that this was effected by the change you made, it has never been mentioned by anyone to this point as being powerful, out-of-balance, broken, etc.. There are no official discussions about this "issue" because to have a discussion, you must start one. Which is what I am attempting to do. 
 
You didn't make the change in #1927 to nerf garynidia so why are you mentioning picking up the pieces? I am attempting to pick up the pieces and game balance around something you unknowingly nerfed massively. You simply overlooked a rutile source and I'm trying to return it back to its original state.

If you did make the change in #1927 to nerf garnydinia please make an arguement as to why you did this.

**Comment by chochem** [comment]
> You have admitted that you didn't know that this was effected by the change you made

I said nothing like that.

> Which is what I am attempting to do.

normally you do that with a ticket, not a PR.

**Comment by chochem** [comment]
>  This was the result of new processing lines for bauxite and ilmenite that yielded much higher amounts.

this is not correct btw. it was the result of chem balance. the rest was balance related in the follow up, but not that part.

**Comment by Eysla** [comment]
If I made a ticket that said change 1 character on this one obscure line that very, very little people know about, how would that change things? The nerf to garnydinia was made on complete accident, why would it be unreasonable to revert this change?

**Comment by Eysla** [comment]
>>You have admitted that you didn't know that this was effected by the change you made

>I said nothing like that.

"I havent seen any ticket discussing this. the andradite recipe was added with no discussion either." I see this as an admission that you were not aware of the change to andradite at the time in which you unknowingly nerfed. This PR has informed you of said nerf.

Either way, andradite and therefore garnydinia were indirectly nerfed by a change made to titaniumtetrachloride #1927, and I believe that the nerf to it should be reverted as it was done unknowingly and was not justified at the time that it was nerfed. Additionally, why does the ratio have to be arbitrarily more terrible if no one was using it in the first place?

**Comment by chochem** [comment]
then you misunderstood me somehow, that quote says nothing like that. not sure how that contributes to anything.

the andradite recipe was added without discussion or even numbers (not great). I adjusted the numbers for it (and various other options too!) in the chem balance PR (based on ticket+discussions) but then based on feedback (more discussions) we chose to improve the ones we want to incentivize slightly above old levels. All while I was providing the numbers.

If you are not even willing to talk about the numbers here, we will probably just need to close this, this is pointless.

**Comment by Eysla** [comment]
I believe the numbers should be 80 ores (calcite+yellow garnet) : 1 titanium, as you stated above as these are the number they were at before they were unknowingly nerfed.

**Comment by chochem** [comment]
gonna close this for now. Just antagonism and attempts to twist my works but no willingness to discuss numbers.

A better place might be a RFC ticket or metadev. We should also discuss how easy or hard we want pre-moon titanium sources to be in general including alu gravel, andradite, yellow garnet/calcite, bees, crops. not just one specific one.

**Comment by Alastors** [comment]
@chochem I don't think Eysla is being quite confrontational enough to justify straight closing the ticket, plus likewise, they do make a decent point that reverting part of a change that happened because of an unintentional accident is fair. Though they aren't the only one being antagonistic.

I have to point out, a ticket isn't absolutely required to justify making a pr, there are times in which the pr itself can serve a similar function, 

@Eysla tone down aggression a bit please, and please do provide some values as to how this will affect the balance once again, yes, I know you are reverting part of Chochom's titanium nerf pr to the original state, but that data is important for considering balance changing regardless of if it's a revert or not.

Though ffs guys, take this sort of debate to #github-discussion or #mod-dev

**Comment by xSkewer** [comment]
> gonna close this for now. Just antagonism and attempts to twist my works but no willingness to discuss numbers.
> 
> A better place might be a RFC ticket or metadev. We should also discuss how easy or hard we want pre-moon titanium sources to be in general including alu gravel, andradite, yellow garnet/calcite, bees, crops. not just one specific one.

I don't see a reason to close this PR over "unwillingness to discuss numbers" when Eysla clearly provided numbers that they believe are fair in literally the previous comment. There is also no reason to assume this PR would affect balance, given the fact that this route has been in the game for probably years at this point, and it isn't even the preferred route for challenge runs.

Also, I think it is very funny that you accuse Eysla of "antagonizing you and twisting your words with no numbers" when, in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13901#issuecomment-1627638673, you do exactly this to Piky by claiming he is trying to undermine glowredman's hard work because of some non-existent personal grudge as evidenced by your comments on the PR and discord:
image
https://discord.com/channels/181078474394566657/224191655375273985/1127534646935175258
As far as I can tell, you never provided a single number to prove your argument here and never bothered trying to. Seems like you only want people to provide numbers when it fits your narrative. Given this, as well as other discussions I have seen regarding skips for progression intended to be worse than the main route, I believe you have a clear bias against these additions based on some nonsense idea that any new skips that are added will immediately break the game. In reality, these skips very rarely, if ever, break progression so badly that they become the preferred route for these resources. 

Put aside your biases and stop trying to hide behind "game balance" when you, as a fairly old player, should know that just about nobody was using this route for rutile before the chem rebalance or new lines.

**Comment by Eysla** [comment]
The 300% increase of rutile required, from 1 rutile to 3, to produce 1 ingot of Ti worth of titaniumtetrachloride in #1927 (see PR as to why this change was made) and incidently nerfed all other rutile sources by exactly a factor of 3, specifically those that are utilized in no rocket runs and can be made pre-moon. See pre-nerf and post-nerf below.

prenerf
postnerf

One such source is centrifuging andradite. It can be found as a ore vein that has a high likelyhood of not generating due to it being 40 weight and generating at y=150-200. Even upon finding such a vein, it only generates sporadically throughout. The primary way of sourcing andradite would be centrifuging yellow garnet, obtained via garnydinia (IC2 Crop). Yellow garnet centrifuges into andradite at a 16:5 ratio, a singular garnydinia crop will produce on average 11.4714734 yellow garnet drops per hour (per [https://cubic-chariot.github.io/ic2crops/](url) assuming crop stats of 20gr/30ga/0re in a grasslands biome that is not being watered or fertilized. Which equates to 3.5848354 andradite per hour per crop. See below.

gph
((4 * 0.5633) + (2.0520) + (0.2874 * 0.10) + (0.2874 * 0.11) + (0.2874 * 0.5 * 1) + (5.5698) + (2.6383/4) + (0.4397) + (2.6383 / 9))

Each operation when centrifuging andradite has a 2% chance of yielding a single rutile dust, given our andradite per hour we can find that a single garnydinia crop can produce 0.0716967 rutile per hour per crop. A full LV crop manager (+5% extra drops chance) has 116 farmland available to use after accounting for water and the crop manager itself. So, given rutile per hour per crop we can do ((0.0716967 * (121 - 5)) * 1.05) to find that a LV crop manager can produce up to 8.7326590 rutile per hour per crop manager. 

With all of this in mind, the current yield of titanium per hour (8.7326590 / 3) is 2.9108863 via garnydinia processing per crop manager. With my proposal, it would bring the titanium production back up to 8.7326590 per hour. Which is still miles and miles slower than other rutile sources and likely doesn't compete with any pre-moon sources.

**Comment by Dream-Master** [comment]
If you ask me i would forbid the "no rocket run" and pre moon titanium processing but sure there are players outside like this play style. Sure lets talk about balance if it need a change.

**Comment by Dream-Master** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13995

**Comment by chochem** [comment]
> @chochem I don't think Eysla is being quite confrontational enough to justify straight closing the ticket, plus likewise, they do make a decent point that reverting part of a change that happened because of an unintentional accident is fair. Though they aren't the only one being antagonistic.
> 

there was no unintentional accident. they are just repeating that lie everywhere again and again until someone believes it.

This is a new addition that glod did sneak in without any discussion (while my PR had dev discussion). This didnt exist at all a year ago.

There is a more fundamental discussion about the availability or ease of titanium pre-moon in general I think we need to have but we both seem to agree this is not the place.

Well dream doesnt seem to like in in general. I dont fully agree with that myself but hes the boss and my take is also that no-moon titanium has got to be hard and slow.

(the deleted comment was just offtopic drama)

**Review by Runakai1 — APPROVED** [review]
This PR has been nothing short of a mess, and I would like to apologize for the issues stemming from the GTNH Devs' side. There have been instances of censorship and unnecessary uncivilized conversation. Using the argument of providing numbers to justify decisions is inappropriate, especially when the ratio was already given. Despite that, Eysla presented compelling arguments that swayed my vote in favor of this change. It should not be mandatory to have a formal ticket for discussing changes; open discussion should be encouraged.

Considering that this PR was relatively small, one could argue that there was not much unnecessary effort put into it, and the rejection risk would have been low even without a ticket.

Based on my observation, I am going to approve this PR. The adjustments made on April 27, just before the new PR-Rules, mainly focused on popular routes. I see no valid reason to dismiss this PR. If Dream decides to reject it, then it will be rejected. However, I strongly believe that GTNH thrives through its community and alternative routes, and imposing strict prohibitions on that front is not acceptable to me. Of course this still needs admin approval to be merged as it is a balance PR.

In summary, this PR is good, but there have been issues with managing this, leading to unnecessary drama.

**Review by Dream-Master — APPROVED** [review]
ok vote say yes but if i find out its to much of a buff i revert it :trollface:

**Comment by Alastors** [comment]
> ok vote say yes but if i find out its to much of a buff i revert it :trollface:

It’s not a buff, it’s a revert of something chochom apparently did without telling us he was according to him, if any thing it’s a revert on grounds of Code of Conduct violations

**Comment by chochem** [comment]
thats just not correct. my changes were all discussed and agreed

**Comment by xSkewer** [comment]
Please point us to your discussion of the change to Andradite then because it would be relevant to this PR and I haven't seen any mention of it until now.
